<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tblQuanLyXe">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>501</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>IDXE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Họ Tên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Biển số xe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phương tiện</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="txtMaPhuongTien">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>ID Phương tiện</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSua">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThem">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Thêm </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTenChuXe">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Họ Tên</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBienSoXe">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Biển số xe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoa">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLoaiXe">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>OTo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XeMay</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XeDap</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txttukhoa">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Tìm Kiếm Phương Tiện</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btntimkiem">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm Kiếm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
